moskito-frontend/src/components/Campsitelist/CampsiteList.js

```diff
@@ -0,0 +1,18 @@
+
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import CampsiteListeItem from './CampsiteListItem'
```

Comment:
```suggestion
import CampsiteListItem from './CampsiteListItem'
```

small typo in your import as well as below, when you are using this component

Reply:
absolutely